channels/lts.env

```diff
@@ -1,34 +1,45 @@
-WMDE_RELEASE_VERSION=wmde.11
+# shellcheck disable=2034 # unused variable
+WIKIBASE_SUITE_RELEASE_MAJOR_VERSION=2
+WIKIBASE_SUITE_RELEASE_MINOR_VERSION=0
+WIKIBASE_SUITE_RELEASE_PATCH_VERSION=0
+WIKIBASE_SUITE_RELEASE_PRERELEASE_VERSION=pre.1
+
 WIKIBASE_BRANCH_NAME=REL1_39
 MEDIAWIKI_BRANCH_NAME=REL1_39
 
 # WDQS_VERSION updated from 0.3.118, 0.3.121 is latest release on 01/03/2023
 WDQS_VERSION=0.3.121
 # Head of master on 01/03/2023
 WDQS_FRONTEND_COMMIT_HASH=05b621341894991143a3e89105a50c996062c33d
-# Head of master on 01/03/2023
+# Head of master on 2023/04/15 as per https://github.com/magnusmanske/quickstatements/commits/master
 QUICKSTATEMENTS_COMMIT_HASH=a05157d54a387636a73613b60a3119616882d749
-# Head of master on 01/03/2023
+# Head of master on 01/03/2023 (last commit is from 2023-01-17 as per https://bitbucket.org/magnusmanske/magnustools/src/master/)
 MAGNUSTOOLS_COMMIT_HASH=51a32efcfb7113ac682b8d9a6c30106f5ec106e9
-# 1.0.2 latest release on 01/03/2023 (main branch README states 1.39 compatability)
+# 1.0.2 latest release on 04/10/2022 (main branch README states 1.39 compatability) https://github.com/ProfessionalWiki/WikibaseLocalMedia/tree/1.0.2#platform-requirements
 WIKIBASELOCALMEDIA_COMMIT_HASH=66fd4e622d9e39125a68fe67f49c9657a0aa35dc
-# 2.0.1 latest release on 01/03/2023 (main branch README states 1.39 compatability)
+# 2.0.1 latest release on 26/03/2022 (main branch README states 1.39 compatability) https://github.com/ProfessionalWiki/WikibaseEdtf/blob/master/README.md#installation
```

Comment:
Please use a consistent date format - 2023/04/15 and 04/10/2022 are both used here

Reply:
Good catch, thanks, done.